addon/services/gwr.js

```diff
@@ -22,6 +23,12 @@ export default class GwrService extends Service {
     return `${this.municipality}00`;
   }
 
+  get housingStatLink() {
```

Comment:
IMO this would be fine as a helper since its not really gwr api specific, it just seems a bit out of place here.